charts/prometheus-federator/0.1.0/charts/helmProjectOperator/values.yaml

```diff
@@ -177,7 +177,7 @@ debugLevel: 0
 cleanup:
   image:
     repository: rancher/shell
-    tag: v0.1.18-rc8
+    tag: v0.1.14
```

Comment:
This change was introduced in https://github.com/rancher/charts/pull/1991, but we didn't bump the chart version. Since we don't want to do an OOB release, this change was removed in this PR in favor of a brand new chart.